js/mmoda.instrument.js

```diff
@@ -2142,7 +2145,9 @@ function panel_title(srcname, param) {
 
     var session_id = data.session_id;
     var session_job_ids = '<div>Session ID : ' + session_id + '</div><div>Job ID : ' + job_id + '</div>';
-    $('#' + panel_ids.panel_id).data("log", session_job_ids + $('.modal-body', '#ldialog').html());
+    // width: 75%;
+    $('#' + panel_ids.panel_id).css('width', '65%');
```

Comment:
It would be better to move css changes to mmoda.css

Reply:
in the css file now